bless/config/bless_deploy_example.cfg

```diff
@@ -1,13 +1,15 @@
 # This section and its options are optional
 [Bless Options]
 # Number of seconds +/- the issued time for the certificate to be valid
-certificate_validity_window_seconds = 120
+certificate_validity_seconds = 120
```

Comment:
We have certificate validity set for 7 days but valid_before and valid_after could work as well. I could add them in a separate PR?